internal/cmd/grants.go

```diff
@@ -230,6 +231,10 @@ func addGrant(cli *CLI, cmdOptions grantsCmdOptions) error {
 		}
 	}
 
+	if err := checkResourcesPrivileges(client, cmdOptions.Destination, cmdOptions.Role); err != nil {
```

Comment:
This is a nice place to add a `force` flag to ignore nonexistent destinations, resources, or roles